### PR TITLE
Add font types to asset filter (~a)

### PR DESCRIPTION
**Pull request by elespike**
#### Description

Add 2 font MIME type patterns to the asset filter (~a) to both `mitmproxy` and `mitmweb`:
- `font/*`
- `application/font-*`

#### Checklist

 - [x] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

### CODE REVIEW

File: mitmproxy/flowfilter.py [hunk 1]
```diff
@@ -15,7 +15,9 @@
                         application/javascript
                         text/css
                         image/*
-                        application/x-shockwave-flash
+                        application/x-shockwave-flash ?
```
Comment (elespike) [inline mitmproxy/flowfilter.py:18]: Should flash be included in the patterns too? It currently isn't.

**Comment by elespike** [comment]
Looks like docs and mitmweb's JS files won't need manual changes, correct?

Cool to see you use pyparsing! I've used it in one of my tools as well =)

File: mitmproxy/flowfilter.py (hunk 1 above)
Comment (mhils) [reply mitmproxy/flowfilter.py]: Thanks for catching this leftover. Flash nowadays would be a rather interesting find than anything else, we removed it from the asset list a while ago.

**Review by mhils — APPROVED** [review]
Thanks for the excellent PR. I really appreciate that you properly looked into this, found the Flash leftover, and also updated the tests. 🍰 

I have reverted the test change as it IMO does more harm than good. The current test is short and explicit, which are desirable properties. There is no benefit in automatically checking more regexes in what is more of a self-fulfilling prophecy. I appreciate you looking into this anyway, hope my reasoning makes sense. 😃

**Comment by elespike** [comment]
Makes sense, and my pleasure =)

**Pull request merged**